ctapipe/image/hillas.py

```diff
@@ -165,6 +165,28 @@ def hillas_parameters(geom, image):
 
         m4_long = np.average(longitudinal ** 4, weights=image)
         kurtosis_long = m4_long / length ** 4
+    
+    """
+    Compute Hillas parameters uncertainties. 
+    Implementation based on W. Wittek. 2002. Image parameters. MAGIC-TDAS internal note 02-03. 
+    http://wwwmagic.mppmu.mpg.de/documents/tdasnotes.html    
```

Comment:
These are internal magic documents.

Reply:
We can't reference to MAGIC internal documents ?

Reply:
We can and should if there is now open source, but most people won't be able to access them, so at least a comment that these are internal magic documents not generally accessible should be added.

Reply:
Comment about the MAGIC internal note added